src/eos-dropbox-app

```diff
@@ -64,18 +64,32 @@ def get_dropbox_directory():
     with open(config_path, 'r') as config:
         logging.info('Found Dropbox configuration at {}'.format(config_path))
 
+        user_config = None
+        account_type = 'Unknown'
         data = config.read()
+
         try:
             json_data = json.loads(data)
-            try:
-                dropbox_dir = os.path.expanduser(json_data['personal']['path'])
-                logging.info('Found configured Dropbox directory at {}'.format(dropbox_dir))
-            except KeyError:
-                logging.warning('Could not find Dropbox directory in user\'s configuration')
 
+            if 'personal' in json_data:
```

Comment:
I would prefer not to assume that the only account types are "personal" or "business".  I don't see why we should care what the account type is.  As long as a path is provided, we should just use it.